libraries/3rdparty/mbedtls_utils/README.md

```diff
@@ -0,0 +1,6 @@
+## Background
+This folder contains the mbedtls_utils.c file. 
+
+The convert_pem_to_der() function defined in mbedtls_utils.c is used by the pkcs11 tests and the dev_mode_key_provisioning demo. This function is copied from the pem2der.c file that can be found in the mbedtls/programs/util/ directory of the mbed tls library. Copying this function is required since the pem2der.c file has a main function and could not be used.
```

Comment:
"PKCS #11 tests"

Reply:
Fixed in [this commit](https://github.com/aws/amazon-freertos/pull/1881/commits/55e856c5319e940ed086585cd66e7efb6d297a94)